src/RecordTask.cc

```diff
@@ -1724,7 +1724,7 @@ bool RecordTask::record_remote_by_local_map(remote_ptr<void> addr,
   return false;
 }
 
-void RecordTask::record_remote(remote_ptr<void> addr, ssize_t num_bytes) {
+void RecordTask::record_remote(remote_ptr<void> addr, ssize_t num_bytes, bool sizeIsConservative) {
```

Comment:
We use snake case, so `size_is_conservative`.

But for public APIs it's better to use a two-value enum than a bool parameter --- more readable.

Reply:
I wasn't 100% sure what a good name for this would be. `MemWriteSizeValidaiton` seems a bit wordy, but there it is.